### ISSUE TITLE
Appending caption text for screen readers

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#464]
**Is your feature request related to a problem? Please describe.**
Without appending text that denotes a caption, a screen reader user won't know that text being read out is the caption of an image. 

**Describe the solution you'd like**
We should append `Image caption, ` to the `figcaption`. This text should be visually hidden.

**Additional Context**
Similar text will also be appended to a Video, which this is out of scope for this issue.

**Comment by sareh** [comment]
Is there a reason why visually-hidden text `image caption` is being added to an element that is already semantically a caption? I understand that `<figcaption>` is not well-supported in many screenreaders.
Can we address this issue in a different way?

Linking the `<figcaption>`  to the image with an `aria-labelledby` attribute. 
e.g. using the example from the source listed below:
 ```
<figure>
    <img src="0123456789.jpg" alt="" aria-labelledby="figcaptionlabel1">
    <figcaption id="figcaptionlabel1">Balboa Park, San Diego, is a popuplar destination for tourists and locals alike.</figcaption>
</figure>
```
https://dequeuniversity.com/presentations/html5-nfb/figure-figcaption

**Comment by pjlee11** [comment]
@greenc05 could you confirm how we should go forward with this following @sareh's comment. I'm assuming by having offscreen text it will work universally for all screen reader variations however this also requires translating the offscreen text for the various world service sites. Whereas using `aria-labelledby` as Sareh suggests would remove the need for translation, however I'm not aware of the support of aria-labelledby across different screen readers.

**Comment by greenc05** [comment]
Ok so, figcaption is only acknowledged by VoiceOver Mac, none of our other supported screen readers acknowledge this (same goes for figure), hence the reason for providing visually hidden text to announced to screen reader users that this is a caption and not part of the article body copy text.

I see looking at the Deque example from @sareh (thanks you Sareh) that the aria-labelledby implementation is only supported by NVDA and Firefox, though we should test this. Can we create a test example and test support for this with our supported AT @pjlee11 ? We can then compare the assistive technology results and compare the UX with the visually hidden span approach.
